packages/o-spreadsheet-engine/src/types/chart/chart.ts

```diff
@@ -103,8 +103,19 @@ export interface DatasetDesign {
   readonly label?: string;
 }
 
+export type AxisScaleType = "linear" | "logarithmic";
+
+export interface AxisGridDesign {
+  readonly major?: boolean;
+  readonly minor?: boolean;
+}
+
 export interface AxisDesign {
   readonly title?: TitleDesign;
+  readonly min?: number;
+  readonly max?: number;
+  readonly scaleType?: AxisScaleType;
+  readonly grid?: AxisGridDesign;
```

Comment:
nitpick: probably more clear to have a boolean `showMajor/minorGridLine` rather than `design.grid.major`

Reply:
Then what about a showGrid that can be "major", "minor", "none" or "both" ?